src/components/Rockets.jsx

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reservation } from '../redux/Rockets/rocketsSlice';
+
+const Rocket = (props) => {
+  const dispatch = useDispatch();
+
+  const {
+    id, name, disc, image, Reservation, reservationState,
+  } = props;
+  /* eslint-disable */
+  return (
+    <div className="rocketC" key={id}>
+      <img className="rocketImg" src={image} alt={name} />
+      <div className="contentC">
+        <h4 className="rocektName">{name}</h4>
+        <div className="rocketDesc">{disc}</div>
+        <div type="button" onClick={() => { dispatch(reservation(Reservation)); }}>{reservationState}</div>
+      </div>
+    </div>
```

Comment:
- [ ] @Team great work on your rockets list, however, the  first image is not getting displayed and the reserve button color and reserved word are not similar to the design given, see the comparison below

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Elhambasir/group-project-space-travelers-hub/assets/61408860/73df62cc-dc9b-4239-8225-e23f15fa5e73)  | ![image](https://github.com/Elhambasir/group-project-space-travelers-hub/assets/61408860/f7caefc1-bf31-40ad-af55-0d1aa96160dd)